tests/integration/frameworks/test_tensorflow_v2_unit.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
```

Comment:
@larme  I added this to test out tensorflow container, would love to hear feedback.